airdrop/api/grant.ts

```diff
@@ -1,17 +1,37 @@
-import type { VercelRequest, VercelResponse } from '@vercel/node';
+import type { VercelRequest, VercelResponse } from "@vercel/node";
 
-const grant = require('grant').vercel({
-  config: grantConfig(), session: {secret: 'grant'}
-})
-
-
-function grantConfig() {
-  const config = require('./config/grant.json');
+function getConfig() {
+  const config = {
+    defaults: {
+      origin: "https://docs.crab.network",
+      transport: "querystring",
+      state: true,
+    },
+    github: {
+      key: "{{ github_oauth_app_key }}",
+      secret: "{{ github_oauth_app_secret }}",
+      scope: [],
+      callback: "/api/crab/authorization",
+      overrides: {
+        crab: {
+          callback: "/api/crab/authorization",
+        },
+        pangolin: {
+          callback: "/api/pangolin/authorization",
+        },
+      },
+    },
+  };
   config.github.key = process.env.GITHUB_OAUTH_APP_KEY;
-  config.github.secret = process.env.GITHUB_OAUTH_APP_SECRET;
+  config.github.secret = process.env.GITHUB_OAUTH_APP_KEY;
```

Comment:
I think there used the wrong variable. should be using `process.env.GITHUB_OAUTH_APP_SECRET`

Reply:
done